model/VocabularyConfig.php

```diff
@@ -178,12 +178,21 @@ public function getTitle($lang = null)
     }
 
     /**
-     * Returns a boolean value set in the config.ttl config.
-     * @return boolean
+     * Returns the sorting strategy for notation codes set in the config.ttl
+     * config: either "decimal" (default), "natural", or null if
+     * sorting by notations is disabled.
+     * @return string|bool
      */
-    public function sortByNotation()
+    public function getSortByNotation(): ?string
```

Comment:
I renamed the method for better consistency with other getters, and redefined its return value.